tests/test_vault.yml

```diff
@@ -61,6 +64,18 @@
               ip_sans: "127.0.0.1"
               alt_names: "example.com"
               exclude_cn_from_sans: true
+    - name: Configure PKI
+      import_role:
+        name: vault_pki
+      vars:
+        vault_token: "{{ vault_keys.root_token }}"
+        vault_pki_root_ca_name: "OS-TLS-ROOT"
+        vault_pki_intermediate_ca_name: "OS-TLS-INT"
+        vault_pki_root_create: False
+        vault_pki_intermediate_create: False
+        vault_pki_generate_certificates: True
+        vault_pki_write_pem_bundle: True
+        vault_pki_certificate_subject:
           - role: 'ServerCert'
```

Comment:
Would we ever want to generate some as PEM bundles and some as separate cert & key in a single run?

Reply:
Probably makes sense, will rework.

Reply:
I don't think it needs reworked. 
I would consider whats done in the test_vault.yml as a single run ?